starlark/src/stdlib/list.rs

```diff
@@ -145,7 +145,8 @@ starlark_module! {global =>
     /// x.index("a", -2) == 5  # bananA
     /// # )"#).unwrap());
     /// ```
-    list.index(this, #needle, #start = 0, #end = None) {
+    list.index(this, #needle, #start = 0, ?#end) {
+        let end = end.unwrap_or_else(|| Value::new(None));
```

Comment:
Why not keep the `= None` here?

Reply:
You are right, `None` is allowed here by Starlark spec: https://github.com/bazelbuild/starlark/blob/master/spec.md#listindex

Reply:
My mistake.